#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_docdir-base.dpatch by Jack Moffitt <jack@xiph.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Change docdirbase to be consistent with Debian policy

@DPATCH@
diff -Nur ipython-0.7.0.orig/setup.py ipython-0.7.0/setup.py
--- ipython-0.7.0.orig/setup.py	2005-12-25 00:46:14.000000000 +0000
+++ ipython-0.7.0/setup.py	2006-01-21 19:29:34.000000000 +0000
@@ -99,7 +99,7 @@
 # Note that http://www.redbrick.dcu.ie/~noel/distutils.html, ex. 2/3, contain
 # information on how to do this more cleanly once python 2.4 can be assumed.
 # Thanks to Noel for the tip.
-docdirbase  = 'share/doc/ipython-%s' % version
+docdirbase  = 'share/doc/ipython'
 manpagebase = 'share/man/man1'
 
 # We only need to exclude from this things NOT already excluded in the
